source/features/clean-conversations.css

```diff
@@ -1,8 +1,14 @@
-/* Remove user avatar from new comment form on threads */
-.review-thread-reply .avatar {
+/* Remove user avatar from new comment form */
+:is(.review-thread-reply, .timeline-new-comment) .avatar {
```

Comment:
- Despite the name, `.review-thread-reply` is also used in Discussions thread reply (could it be that they are review comments in disguise? 🤔)
- `.timeline-new-comment` works on both conversations and commits page